src/schema/v2/artist/helpers.ts

```diff
@@ -190,3 +195,21 @@ export const getAuctionRecord = async (artist, auctionLotsLoader) => {
 
   return highAuctionRecord
 }
+
+const getRecentShow = (artist): string[] => {
+  // dd/mm/yyy|slug|Group or Solo|Show Title
+  const entities = splitEntities(artist.recent_show)
+  if (!entities) return []
+
+  const date = entities[0]
+  const show = entities[entities.length - 1]
+
+  const showDate = new Date(date)
+  const today = new Date()
+  const threeYearsAgo = new Date()
+
+  threeYearsAgo.setFullYear(today.getFullYear() - 3)
+
+  if (showDate < threeYearsAgo) return []
+  return [show]
```

Comment:
Empty array and array with show both are valid result for [getEntities](https://github.com/artsy/metaphysics/blob/510f1be086baff907ec2b865ddb483ce28ad9ad9/src/schema/v2/artist/helpers.ts#L150-L151), this badge will be shown in both cases. I used `null` [value instead of empty array](https://github.com/artsy/metaphysics/pull/5115#discussion_r1254587168).